schema/src/lib.rs

```diff
@@ -359,7 +359,7 @@ impl<T: IntoSchema, const L: usize> IntoSchema for [T; L] {
         let _ = map.entry(Self::type_name()).or_insert_with(|| {
             Metadata::Array(ArrayMeta {
                 ty: T::type_name(),
-                len: L,
+                len: L as u64,
```

Comment:
Also think, that it's better to use `expect()`

```rust
try_from(L).expect("`usize` always fits in `u64`")
```

Reply:
It really is better, unfortunately Rust still supports some of C's cast-related functionality, even though it doesn't actually need most of it.